rpm/droidmedia.spec

```diff
@@ -46,7 +46,12 @@ mv droidmedia* droidmedia
 popd
 
 %build
+
+%if "%{?BOARD_QTI_CAMERA_32BIT_ONLY:%{BOARD_QTI_CAMERA_32BIT_ONLY}}%{!?BOARD_QTI_CAMERA_32BIT_ONLY:0}" == "true" 
```

Comment:
I think droidmedia builds in an androidy environment, with droid-bin-src-full etc. It's gst-droid that just uses mb2.